api/v1alpha4/openstackcluster_webhook.go

```diff
@@ -105,6 +105,12 @@ func (r *OpenStackCluster) ValidateUpdate(oldRaw runtime.Object) error {
 		r.Spec.ControlPlaneEndpoint = clusterv1.APIEndpoint{}
 	}
 
+	// Allow changes to the bastion spec only if no bastion host is deployed (i.e. Spec.Bastion.Enabled=false).
+	if old.Status.Bastion == nil {
+		r.Spec.Bastion = &Bastion{}
+		old.Spec.Bastion = &Bastion{}
+	}
```

Comment:
Won't this prevent setting Enabled=false once the bastion is deployed? i.e. we can never undeploy it in order to make changes?

Reply:
Good catch 🙂 fixed it

Reply:
I must have lost sight of the flag somehow 😅